pylint.cfg

```diff
@@ -1,2 +1,3 @@
 [MESSAGES CONTROL]
-disable=C0330,R0903,W1202,W1203
+disable=C0330,R0903,W1202,W1203,R0913,R0902
```

Comment:
If you include these here, you can remove the corresponding pylint directives in the individual modules, e.g., in `htsinfer.htsinfer`, line 74